docs/en/install.md

```diff
@@ -16,19 +16,29 @@ To uninstall Zettlr, simply run the Uninstall.exe either from the directory itse
 
 ## macOS (10.11 or newer)
 
-To install Zettlr on macOS, simply download the dmg-file and mount it by double-clicking it. Then drag the Zettlr icon into your Applications directory and you're done!
+To install Zettlr on macOS, simply download the dmg-file from our [download page](https://www.zettlr.com/download) and mount it by double-clicking it. Then drag the Zettlr icon into your Applications directory and you're done!
 
 To uninstall Zettlr, simply remove the Zettlr.app from your Applications directory. If you want to completely remove all data associated with the app, also remove the directory `/Users/<your-user-name>/Library/Application Support/Zettlr`.
 
 > You can also install Zettlr using [Homebrew](https://formulae.brew.sh/cask/zettlr): `$ brew install --cask zettlr`
 
-## Linux (Debian 8/Ubuntu 12.04/Fedora 24 or newer)
+## Linux
 
-There are prebuilt `deb` and `rpm` packages as well as `AppImage` bundles for Linux systems. Simply install the package on your system according to which of these you have downloaded. For installing AppImage bundles, please [refer to this guide on how to install them](https://appimage.org/).
+### Debian/Ubuntu/Fedora and derivatives
+
+There are prebuilt `deb` (Debian 8/Ubuntu 12.04 or newer) and `rpm` (Fedora 24 or newer) packages. You can install them using the respective package manager or download them from our [download page](https://www.zettlr.com/download).
 
 To uninstall, follow the usual steps it takes to remove a package (usually through the graphical installer application or via `dpkg`). If you also want to remove all data associated with the app, also delete the `/home/<your-user-name>/.config/Zettlr` directory.
 
-> There is a community-maintained package for Arch Linux available. [You can find it on the official AUR repositories](https://aur.archlinux.org/packages/zettlr-bin/). Please note that this package is community-maintained and we do not take any responsibility in its stability, safety or offered version.
+### Arch Linux
+
+There are two community-maintained packages available for Arch Linux. The [zettlr](https://archlinux.org/packages/?name=zettlr) package contains the latest release; the [zettlr-git](https://aur.archlinux.org/packages/zettlr-git/) AUR package represents the latest commit from GitHub. You can find more information in the [Arch wiki article](https://wiki.archlinux.org/title/Zettlr) about Zettlr.
```

Comment:
Is `?name=zettlr` vis a vis `/packages/zettlr/git` intended?

Reply:
Its intended, because the `zettlr` package is part of a different repository than `zettlr-git` was (Non-AUR vs AUR). The links of these two kinds of repos have a different structure.

If we wanted to link directly to the package instead of the search, we'd have to specify an architecture. Since the package seems to be available only for x86_64 currently, we could link here: https://archlinux.org/packages/community/x86_64/zettlr/
However, as the package might become available for other architectures such as ARM or RISC-V in the future, I think linking to the search is better.

Reply:
Alright, I just wanted to confirm that this was not just a mistake! I agree that a more generic, platform-independent link is the way to go. Thanks for confirming!